README.md

```diff
@@ -48,34 +48,33 @@ Each MD page in `/faq/` has one or more taxonomy defined in header. E.g.:
 
 ```yaml
 taxonomies:
-  faq: ["Android"]
+  faq: ["app"]
 ```
 
 Zola collects all such taxonomies:
 
-| File                 | Taxonomy key | Taxonomy value |
-| -------------------- | ------------ | -------------- |
-| android-lags.md      | `faq`        | `Android`      |
-| android-logs.md      | `faq`        | `Android`      |
-| general-team.md      | `faq`        | `General`      |
-| general-bugreport.md | `faq`        | `General`      |
-| ios-versions.md      | `faq`        | `iOS`          |
-| map-longtap.md       | `faq`        | `Map`          |
-| ...                  | ...          | ...            |
+| File                                    | Taxonomy key | Taxonomy value |
+|-----------------------------------------| ------------ |----------------|
+| map/can-find-position/index.md          | `faq`        | `map`          |
+| map/search-cannot-find-a-place/index.md | `faq`        | `map`          |
+| editing/map-errors/index.md             | `faq`        | `editing`      |
+| app/crash/index.md                      | `faq`        | `app`          |
+| text-to-speech-android-tts/index.md     | `faq`        | `tts`          |
+| ...                                     | ...          | ...            |
 
-After that Zola gets all values for `faq` taxonomy: `[General, Android, iOS, Map, ...]`. And generates pages:
+After that Zola gets all values for `faq` taxonomy: `[app, map, editing, tts, ...]`. And generates pages:
 
 - For key `/faq/` with the list of values (see `templates/faq/list.html`)
-- For each value `/faq/general`, `/faq/android`, etc. with the list of questions (see `templates/faq/single.html`)
+- For each value `/faq/app`, `/faq/map`, etc. with the list of questions (see `templates/faq/single.html`)
```

Comment:
Does it make sense to mention that for other languages taxonomies should be translated for better SEO?

Reply:
Added translated taxonomies samples: `/es/faq/aplicacion/` and `/de/faq/karte/`